src/components/btn.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import "../App.css";
+
+class Btn extends Component {
+  render() {
+    return (
+      <div className="btn-container" onClick={this.props.onClick}>
```

Comment:
just wondering - why dont you use `<button>` tag?

Reply:
because button container has own built-in style that i have to disable in css,
in the meanwhile it is easier to write my own button from scratch.